babel.config.js

```diff
@@ -1,3 +1,4 @@
+//You will require babel plugin to remove the console.log from production build.
```

Comment:
Reword to `The babel plugin is needed to remove the console.log from production builds.`